testing/__init__.py

```diff
@@ -28,6 +28,13 @@ def assert_adhoc_turn_created(self, count=1):
         after_turn_count = AdHocTurn.objects.count()
         self.assertEqual(after_turn_count, before_turn_count + count)
 
+    @contextmanager
+    def assert_adhoc_turn_not_created(self, count=1):
```

Comment:
Is there a problem with using `with self.assert_adhoc_turn_created(count=0):`

Reply:
A rather large problem, actually. If you look closely, you'll see that parameter, `count`. If you don't look as closely, you won't notice it, you see.

Reply:
Lol, happens to the best of us.